docs/changelog/changelog_en.md

```diff
@@ -1,49 +1,46 @@
-- Created new useCarousel composable holding Carousel logic
-- Minified and purged swiper css files for speed optimization
-- Fluid logo container + max-width and max-height addaptation
-- Fix cls mobile
-- Edit mode toolbar + Json editor Front End
+## v1.7.0 (2024-11-06) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.6.0...v1.7.0" target="_blank" rel="noopener"><b>Overview of all changes</b></a>
 
 ### New
 
-- Clicking "Buy" with an unsaved Address will now display a notification.
-- When an EU country is selected as the shipping country, all EU countries become available as billing options.
-- Added a new request header for configId and added no cache to environment variables.
-- Implement new notification design
-- Adding the ability to have alt text for images.
-- It's now possible to change the font color of the hero banner via a template property.
-- Display product titles without global suffix when available, maintaining backward compatibility.
+- Clicking the **Buy** button with an unsaved address now displays a notification.
+- Selecting an EU country as the shipping country now makes all EU countries available as billing options.
+- Improved the product title display for SEO purposes. Reverts to the default if no product specific title is available.
+- Added tax and shipping note to the wishlist page.
+- Added a new request header for the `configId`.
+- Added the environment variable `NO_CACHE` for disabling caching.
+- The hero banner and media card now support alt text properties for images.
+- Enabled font color customization for the hero banner via a template property.
+- Added a new carousel compoment.
+- Added an edit mode toolbar and JSON editor for the front end. Note that this is a preparatory step. Further functionality will be added in an upcoming version.
 
 ### 🩹 Fixed
 
-- Fixed an issue where increasing quantity over maximum stock will lead to cart being cleared.
-- Removed the "Add to Cart" notification from the item and category pages when the quick checkout modal is not present.
-- Fixed an accessibility issue where the font size was too small.
-- Fixed an issue where product path was not reactive when category was changed.
-- Automatically generate a language file for every active language, not just the default language.
-- Soft login was still shown after successfully authenticating on the order confirmation page.
-- Fixed an issue where manufacturer data was introducing 'name' into structured data instead of 'externalName'.
-- Headlines now use the configured font.
-- Fixed layout shift on category page.
-- The build script failed on Windows because of incompatibilities between file name pattern and operating system. The file name pattern now works on Windows.
+- Fixed an issue where increasing the quantity beyond maximum stock would clear the cart.
+- Removed the "Add to Cart" notification from item and category pages when the quick checkout modal is not present.
```

Comment:
Isn't that controlled by a setting?

Reply:
@Tim-M-S 
quick checkout is not controlled by environment setting, only the timer is.

Reply:
According to Andrei, no.